stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageList.kt

```diff
@@ -202,19 +209,22 @@ public fun Messages(
             }
         }
 
+        val firstVisibleItemIndex = currentListState.firstVisibleItemIndex
+
         when {
             newMessageState == MyOwn -> coroutineScope.launch {
-                currentListState.scrollToItem(5)
+                if (firstVisibleItemIndex > 5) {
```

Comment:
can this `5` be customizable ? 

Since this could be internal SDK thing but was wondering if there is a reason for having such a constant declared as is.

Reply:
This should actually be changed to some form of offset. This was put in initially as a pseudo-random number, but will be updated!